config/locales/notices/en.yml

```
@@ -16,6 +16,10 @@ en:
       use_diff_email:
         link: create a new account
         text_html: Or, %{link} using a different email address.
+    irs_data_use:
+      link: Learn more about our privacy policies
+      text_html: This is a U.S. government service. You consent to the monitoring,
+        recording, and reviewing of your activity using this service. %{link}.
```

I would take the `%{link}` out of here since it's at the end and could just be a second item in the template. I'll make an example suggestion for the block syntax. Then it no longer needs an `_html` suffix

```suggestion
      text: This is a U.S. government service. You consent to the monitoring,
        recording, and reviewing of your activity using this service.
```

---

This comment is no longer relevant